src/onnx/onnx.cpp

```diff
@@ -593,6 +553,56 @@ struct onnx_parser
         }
     }
 
+    template <class Op>
+    void cal_auto_padding_size(node_info info,
+                               Op& op,
+                               std::vector<std::size_t> k_lens,
+                               std::vector<std::size_t> dilation,
+                               const std::vector<std::size_t>& in_lens,
+                               std::vector<int64_t>& paddings)
```

Comment:
It seems like this should be the return value of the function.

Reply:
yes, paddings is the return value of the function.